src/test/java/hw/two/ExercisesHw2.java

```diff
@@ -0,0 +1,136 @@
+                .getText(), "HOME");
+        assertEquals(driver.findElement(By.cssSelector("div.uui-header.dark-gray a[href='contacts.html']"))
+                .getText(), "CONTACT FORM");
+        assertEquals(driver.findElement(By.cssSelector("div.uui-header.dark-gray a[class = 'dropdown-toggle']"))
+                .getText(), "SERVICE");
+        assertEquals(driver.findElement(By.cssSelector("div.uui-header.dark-gray a[href='metals-colors.html']"))
+                .getText(), "METALS & COLORS");
+
+        //7 Checking icons
+        assertTrue(driver.findElement(By.cssSelector(".icons-benefit.icon-practise")).isDisplayed());
+        assertTrue(driver.findElement(By.cssSelector(".icons-benefit.icon-custom")).isDisplayed());
+        assertTrue(driver.findElement(By.cssSelector(".icons-benefit.icon-multi")).isDisplayed());
+        assertTrue(driver.findElement(By.cssSelector(".icons-benefit.icon-base")).isDisplayed());
+
+        //8 Checking text under icons
+        List<WebElement> texts = driver.findElements(By.cssSelector(".benefit-txt"));
+        assertEquals(texts.get(0).getText(), "To include good practices\n"
+                + "and ideas from successful\n"
+                + "EPAM project");
+        assertEquals(texts.get(1).getText(), "To be flexible and\n"
+                + "customizable");
+        assertEquals(texts.get(2).getText(), "To be multiplatform");
+        assertEquals(texts.get(3).getText(), "Already have good base\n"
+                + "(about 20 internal and\n"
+                + "some external projects),\n"
+                + "wish to get more…");
+
+        //9 Frame button is displayed
+        driver.switchTo().frame("frame");
+        assertTrue(driver.findElement(By.cssSelector("#frame-button")).isDisplayed(), "Frame Button is displayed");
+        driver.switchTo().defaultContent();
+
+        //10 Checking 5 items in the left section
+        assertEquals(driver.findElement(By.cssSelector("ul.sidebar-menu.left > li:nth-of-type(1)"))
+                .getText(), "Home");
+        assertEquals(driver.findElement(By.cssSelector("ul.sidebar-menu.left > li:nth-of-type(2)"))
+                .getText(), "Contact form");
+        assertEquals(driver.findElement(By.cssSelector("ul.sidebar-menu.left > li:nth-of-type(3)"))
+                .getText(), "Service");
+        assertEquals(driver.findElement(By.cssSelector("ul.sidebar-menu.left > li:nth-of-type(4)"))
+                .getText(), "Metals & Colors");
+        assertEquals(driver.findElement(By.cssSelector("ul.sidebar-menu.left > li:nth-of-type(5)"))
+                .getText(), "Elements packs");
+
+        driver.findElement(By.cssSelector("div.logout")).click();
+
+    }
+
+    @Test
+    public void exercise2() {
+
+        //3 Assert
+        assertEquals(driver.getTitle(), "Home Page");
+
+        //4 Login
+//        driver.findElement(By.cssSelector("li.dropdown.uui-profile-menu")).click();
```

Comment:
Is there a reason you keep commented line here? Better not to keep commented code if you no longer need it

Reply:
deleted